.editorconfig

```diff
@@ -29,35 +29,41 @@ indent_size = 4
 end_of_line = crlf
 trim_trailing_whitespace = true
 
-dotnet_analyzer_diagnostic.category-Style.severity = error
+# Do not ever set to error, or these cause build failures during development.
+dotnet_analyzer_diagnostic.category-Style.severity = warning
 
 # Organize usings
 dotnet_sort_system_directives_first = true
 dotnet_separate_import_directive_groups = false
 
 file_header_template = Copyright (c) Microsoft Corporation.\nLicensed under the MIT License.
 
-dotnet_diagnostic.IDE0003.severity = error    # this and Me preferences
-dotnet_diagnostic.IDE0007.severity = error    # Use 'var' instead of explicit type
-dotnet_diagnostic.IDE0005.severity = error    # Remove unnecessary using directives
+dotnet_diagnostic.IDE0003.severity = warning    # this and Me preferences
```

Comment:
we don't want "this.", "Me" why change it?

Reply:
this isn't removed. it's moved so all IDE* rules are in one place.
They are also made as a warning to not block development progress.